docs-specs-generator/input/template.tmpl

```diff
@@ -0,0 +1,131 @@
+
+* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic), minimum version 1.19.2
+* Linux distribution [compatible with the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems)
+
+## Install and activate [#install]
+
+To install the {{ .HumanReadableIntegrationName }} integration, choose your setup:
+
+<CollapserGroup>
+  <Collapser
+    id="linux-install"
+    title="Linux installation"
+  >
+    1. Follow the instructions for [installing an integration](/docs/install-integrations-package), using the file name `nri-{{ .IntegrationName }}`.
+    2. Change the directory to the integrations folder:
+
+       ```
+       cd /etc/newrelic-infra/integrations.d
+       ```
+    3. Copy of the sample configuration file:
+
+       ```
+       sudo cp {{ .IntegrationName }}-config.yml.sample {{ .IntegrationName }}-config.yml
+       ```
+    4. Edit the `{{ .IntegrationName }}-config.yml` file as described in the [configuration settings](#config).
+
+    5. [Restart the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/configuration/start-stop-restart-check-infrastructure-agent-status).
+  </Collapser>
+
+  <Collapser
+    id="tarball"
+    title="Tarball installation (advanced)"
+  >
+    You can also [install the integration from a tarball file](/docs/integrations/host-integrations/installation/install-host-integrations-built-new-relic#tarball).
+    This gives you full control over the installation and configuration process.
+  </Collapser>
+</CollapserGroup>
+## Find and use data [#find-and-use]
+
+For more on how to find and use your data, see [Understand integration data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data).
+
+Metrics are attached to the Metric sample and the entities{{- range .Entities }} `{{.EntityType}}`,{{- end}}
+[event types](/docs/using-new-relic/data/understand-data/new-relic-data-types#events-new-relic).
+You can [query this data](/docs/using-new-relic/data/understand-data/query-new-relic-data) for troubleshooting purposes
+or to create custom charts and dashboards.
+
+## Metric data [#metrics]
+{{- range .Entities }}
+
+The following dimensional metrics are captured scraping the exporter and linked to the entity {{ .EntityType }}:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ "{{ width: \"350px\" }}" }}>
+        Name
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Dimensions
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+        {{ range .Metrics }}
+
+    <tr>
+      <td>
+        `{{ .Name }} ({{ .Type }})`
+      </td>
+
+      <td>
+        {{ .Description }}
+      </td>
+
+      <td>
+        {{- range .Dimensions }}
+        - {{ .Name }}
+        {{- end }}
+      </td>
+    </tr>
+
+        {{- end }}
+
+  </tbody>
+</table>
```

Comment:
I'd prefer a Markdown table, they shoudl render fine in the docs.

Reply:
Done in a778f7b9c905a6534a3d139479479f300657a444.